src/components/tooltip/Tooltip.tsx

```diff
@@ -312,6 +313,12 @@ class Tooltip extends React.Component<TooltipProps, State> {
             tetherProps.offset = offset;
         }
 
+        if (theme === TooltipTheme.ERROR) {
```

Comment:
It seems like this could be simplified quite a bit with a ternary.

Reply:
@jstoffan I just updated this PR with your comments